Doc/whatsnew/3.11.rst

```diff
@@ -1278,7 +1278,7 @@ This section covers specific optimizations independent of the
   (Contributed by Stefan Behnel in :gh:`68264`.)
 
 * Resizing lists is streamlined for the common case,
-  speeding up :meth:`list.append` by ≈15%
+  speeding up :meth:`!list.append` by ≈15%
```

Comment:
I would left it as a reminder that we should add a reference to list.append, and not sweep it under cover.